<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Stats View</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="CustomTableView" name="tableData">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>250</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="newDocument"/>
    <addaction name="saveDocument"/>
    <addaction name="openDocument"/>
    <addaction name="saveDocumentAs"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="showTable"/>
    <addaction name="showDiagram"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuInsert">
    <property name="title">
     <string>Insert</string>
    </property>
    <addaction name="actionInsertRow"/>
    <addaction name="actionDeleteRow"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menuInsert"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="newDocument">
   <property name="text">
    <string>New Table</string>
   </property>
  </action>
  <action name="saveDocument">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="openDocument">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="saveDocumentAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="showTable">
   <property name="text">
    <string>Table</string>
   </property>
  </action>
  <action name="showDiagram">
   <property name="text">
    <string>Diagram</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Application</string>
   </property>
  </action>
  <action name="actionInsertRow">
   <property name="text">
    <string>Insert Row</string>
   </property>
  </action>
  <action name="actionDeleteRow">
   <property name="text">
    <string>Delete Row</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomTableView</class>
   <extends>QTableView</extends>
   <header>customtableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
